test/integration/stats_integration_test.cc

```diff
@@ -304,7 +305,8 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
   // We only run the exact test for ipv6 because ipv4 in some cases may allocate a
   // different number of bytes. We still run the approximate test.
   if (ip_version_ != Network::Address::IpVersion::v6) {
-    EXPECT_MEMORY_EQ(m_per_cluster, 45002);
+    // https://github.com/envoyproxy/envoy/issues/12209
```

Comment:
note: https://github.com/envoyproxy/envoy/pull/12035 removed this commented-out section, probably due to a git merge conflict error.

This PR actually does reduce memory usage, and I put the updated values into the comment log, but I re-tested the flakiness and still measured 3 flakes out of 100 runs ,so I commented out the tests again.